src/config/config.cc

```diff
@@ -892,8 +891,8 @@ Status Config::finish() {
   if ((cluster_enabled) && !load_tokens.empty()) {
     return {Status::NotOK, "enabled cluster mode wasn't allowed while the namespace exists"};
   }
-  if ((redis_select_compatible) && !load_tokens.empty()) {
-    return {Status::NotOK, "redis-select-compatible mode wasn't allowed while the namespace exists"};
+  if ((redis_databases > 0) && !load_tokens.empty()) {
```

Comment:
@hanxi You can also check if `cluster_enabled` is enabled here. If enabled, redis database feture should be also disasllowed.